website/source/api/system/auth.html.md

```diff
@@ -194,6 +194,14 @@ can be achieved without `sudo` via `sys/mounts/auth/[auth-path]/tune`._
 - `description` `(string: "")` – Specifies the description of the mount. This
   overrides the current stored value, if any.
 
+- `audit_non_hmac_request_keys` `(string:"")` - Specifies the comma-separated
```

Comment:
Since this is a TypeCommaStringSlice, we may want to document this as a `(list: [])`. We could say the comma-separated string can also be provided as an alternative. Same comment for other fields as well.

Reply:
Oh, yes. Please do that. These types all got changed to strings in the API docs revamp a while back and it's been no end of confusion for users.